### PR TITLE
Added CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by Abhishekgupta204**
### 🤔 What is the nature of this change? / 这个变动的性质是？

- [ ] New feature / 新特性提交
- [ ] Fix bug / bug 修复
- [ ] Style optimization / 样式优化
- [ ] Code style optimization / 代码风格优化
- [ ] Performance optimization / 性能优化
- [ ] Build optimization / 构建优化
- [x] Website, documentation, demo improvements / 网站、文档、Demo 改进
- [ ] Refactor code or style / 重构代码或样式
- [ ] Test related / 测试相关
- [ ] Other / 其他

### 🔗 Related Issue / 相关 Issue

(Describe the source of related requirements, such as the related issue discussion link. Example: close #123, close #456. /描述相关需求的来源，如相关的 issue 讨论链接。 例如：close #123, close #456. )

### 💡 Background or solution / 需求背景和解决方案

(The specific problem solved. / 解决的具体问题。)

### 📝 Changelog / 更新日志

(Describe changes from the user side, and list all potential break changes or other risks. / 从用户角度描述具体变化，以及可能的 breaking change 和其他风险。)

### CODE REVIEW

**Comment by Abhishekgupta204** [comment]
@Umpire2018 I have added the code of conduct to make it easier for future contributors to contribute to this repository.
Please review my pull request and merge it.

**Comment by Umpire2018** [comment]
Hello, thank you for submitting this pull request. We will review it internally within our team so stay tuned.

**Comment by Abhishekgupta204** [comment]
Please make the process a bit faster.

File: CODE_OF_CONDUCT [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (Umpire2018) [inline CODE_OF_CONDUCT:63]: Hi, please replace here with `xagentteam@gmail.com`.

**Comment by Umpire2018** [comment]
I noticed that this PR has not been active for a while. I just wanted to remind you that inactive PRs might be closed if there's no response within a day.

**Pull request closed without merging**